patches/api/0402-Mark-experimental-api-as-such.patch

```diff
@@ -1166,17 +1166,18 @@ index 9f935266f51f4ea5d1193031f52e5327350cc06f..ee3a3bda9a5b9cf3bcabf80ee3c653a2
      TEXT_DISPLAY("text_display", TextDisplay.class, -1),
      /**
 diff --git a/src/main/java/org/bukkit/entity/Sniffer.java b/src/main/java/org/bukkit/entity/Sniffer.java
-index dc640c44660acef8f2d17283426052fe7e9e8ae2..50c953619dc8139f064ba139e75a397325ae8cd4 100644
+index 6509d1ca35ff1cb7a2b31871c642d730b2816f6f..67982a8b2258c476eac76e7dc64bc7f143f63df9 100644
 --- a/src/main/java/org/bukkit/entity/Sniffer.java
 +++ b/src/main/java/org/bukkit/entity/Sniffer.java
-@@ -3,6 +3,7 @@ package org.bukkit.entity;
- /**
-  * Represents a Sniffer.
+@@ -12,7 +12,7 @@ import org.jetbrains.annotations.Nullable;
+  * <b>Note: This entity is part of an experimental feature of Minecraft and
+  * hence subject to change.</b>
   */
-+@org.jetbrains.annotations.ApiStatus.Experimental // Paper - 1.20
+-@ApiStatus.Experimental
++@ApiStatus.Experimental // Paper - 1.20
```

Comment:
does that mean that upstream now has this annotation? if so why do paper need this diff?

Reply:
I put those comments there on all the `Experimental` annotations just to know that its specifically related to the 1.20 update. So not really an issue.